admin/zabbix/Makefile

```diff
@@ -262,7 +264,7 @@ CONFIGURE_ARGS+= \
 	$(if $(CONFIG_ZABBIX_MYSQL),--with-mysql) \
 	$(if $(CONFIG_ZABBIX_POSTGRESQL),--with-postgresql) \
 	$(if $(CONFIG_ZABBIX_SQLITE),--with-sqlite3=$(STAGING_DIR)/usr) \
-	--with-libevent=$(STAGING_DIR)/usr/include/libevent \
+	--with-libevent=$(STAGING_DIR)/usr/include \
 	--with-libpcre=$(STAGING_DIR)/usr/include \
```

Comment:
Can we switch it to use libpcre2?

Reply:
This is already prepared https://github.com/openwrt/packages/pull/22534
I think we should merge this first and then I will do a rebase.

Reply:
Merged, please rebase